pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -295,6 +286,22 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 	return kmaster, nil
 }
 
+// GetPluginConfigFile translates from the master plugin config to a file name containing
```

Comment:
Moving it into pkg/cmd/server/admission/init.go - config is kinda init-related, right? Can be changed if that seems hokey.